aggregator/src/aggregator.rs

```diff
@@ -1033,7 +910,9 @@ impl VdafOps {
         report: Report,
     ) -> Result<(), Arc<Error>>
     where
-        A: vdaf::Aggregator<SEED_SIZE, 16> + Send + Sync + 'static,
+        A: vdaf::Aggregator<SEED_SIZE> + Send + Sync + 'static,
+        for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
+        for<'a> <A::AggregateShare as TryFrom<&'a [u8]>>::Error: Debug,
```

Comment:
I've not been able to fully wrap my head around these type bounds, perhaps someone can check my understanding. In english, they're saying.
- A is vdaf::Aggregator, fair enough.
- The associated type `A::AggregateShare` must be `Into<Vec<u8>>` (TIL you can place bounds on associated types).
- The associated type of Error for the `impl TryFrom<&'a [u8]> for A::AggregateShare` must be Debug?

Correct?